lesson_five_hw/task_two_four.js

```diff
@@ -0,0 +1,9 @@
+
+let originalProposal = 'Привет Мир Тест Первый ntcn тест Тест';
+let brokenDownIntoWordsOriginalProposal = originalProposal.toLowerCase().split(" ");
+let capitalizedOriginalProposal = brokenDownIntoWordsOriginalProposal.map(привет=> {
```

Comment:
Try not to use cyrillic in the names of variables, parameters, functions, classes